README.md

```diff
@@ -76,7 +76,6 @@ If you want to compile it from source, you will need Go 1.7 or later, and the [G
 4. Compile the binary:  
   - Linux/macOS/*nix: `go install -o $GOPATH/bin/akamai .``
   - Windows `go build -o $GOPATH/bin/akamai.exe`
-5. Move the binary (`akamai` or `akamai.exe`) it to your `PATH`
```

Comment:
It might be best to add "5. Ensure that `GOPATH/bin` is in your `PATH`" instead of this?